lib/tests/local_request_private_cookie-issue-368.rs

```diff
@@ -0,0 +1,30 @@
+#![feature(plugin, custom_derive, decl_macro)]
+#![plugin(rocket_codegen)]
+
+extern crate rocket;
+
+use rocket::http::{Cookie, Cookies};
+
+#[get("/")]
+fn return_private_cookie(mut cookies: Cookies) -> Option<String> {
+    match cookies.get_private("cookie_name") {
+        Some(cookie) => Some(cookie.value().into()),
+        None => None,
+    }
+}
+
+mod tests {
+    use super::*;
+    use rocket::local::Client;
+
+    #[test]
+    fn private_cookie_works() {

```

Comment:
Can we make sure that a `Set-Cookie` header isn't sent in the response? This confirms that "original private cookies" is working as expected.